Tests/SparkConnectTests/DataFrameTests.swift

```diff
@@ -484,20 +484,24 @@ struct DataFrameTests {
   @Test
   func checkpoint() async throws {
     let spark = try await SparkSession.builder.getOrCreate()
-    // By default, reliable checkpoint location is required.
-    try await #require(throws: Error.self) {
-      try await spark.range(10).checkpoint()
+    if await spark.version >= "4.0.0" {
```

Comment:
I wonder why skipping Spark 3?

Reply:
Ya, I was also surprised with the failure at the first commit.
`Spark Connect` didn't support this feature until 4.0.0.